BaseTools/Source/C/GenFw/elf_common.h

```diff
@@ -181,6 +182,7 @@ typedef struct {
 #define EM_AARCH64  183  /* ARM 64bit Architecture */
 #define EM_RISCV64  243 /* 64bit RISC-V Architecture */
 #define EM_RISCV    244 /* 32bit RISC-V Architecture */
+#define EM_LOONGARCH64 258 /* LoongArch 64-bit Architecture */

```

Comment:
32-bit LoongArch is also 258, so this should be just EM_LOONGARCH? The `EM_RISCV = 244` is not present in upstream binutils either (RISCV is unconditionally 243 there).

Reply:
Okay, it is really not appropriate to defined with EM_LOONGARCH64, I will refer spelling in binutils and update this and other related files. Thanks.

Reply:
> Okay, it is really not appropriate to defined with EM_LOONGARCH64, I will refer spelling in binutils and update this and other related files. Thanks.

I checkout the code, EDK II BastTools distinguishes 32bit and 64bit, is it better to define two? Such as:
EM_LOONGARCH32 258
EM_LOONGARCH64 258
@ChenghuaXu Can you give some recommendation?
Refer EDKII code:https://github.com/tianocore/edk2/blob/master/BaseTools/Source/C/GenFw/Elf32Convert.c and https://github.com/tianocore/edk2/blob/master/BaseTools/Source/C/GenFw/Elf64Convert.c

Reply:
The newest commit solves this issue.